src/main/java/no/nav/veilarbaktivitet/service/AktivitetAppService.java

```diff
@@ -101,6 +101,34 @@ public AktivitetData opprettNyAktivitet(Person ident, AktivitetData aktivitetDat
         return this.hentAktivitet(id); // this is done because of KVP
     }
 
+    @Transactional
+    public AktivitetData oppdaterSvarPaaOmCvSkalDeles(AktivitetData aktivitetData, boolean erEksternBruker) {
```

Comment:
kan denne gå en annen fil? i mappen for deling av cv?

Reply:
Ja, det syns jeg. En egen fil i mappa som dere jobber med i den andre pr'en :)